runtime/doc/windows.txt

```diff
@@ -324,9 +324,9 @@ CTRL-W CTRL-C						*CTRL-W_CTRL-C*
 		screen. For {count} see |:quit|.
 
 		The buffer becomes hidden (unless there is another window
-		editing it or 'bufhidden' is `unload` or `delete`). If the
-		window is the last one in the current tab page the tab page is
-		closed. |tab-page|
+		editing it or 'bufhidden' is `unload`, `delete` or `wipe`).
+		If the window is the last one in the current tab page the tab
+		page is closed.  |tab-page| 
```

Comment:
Not your fault, as it was in the original patch, but let's keep the formatting we used and remove the two spaces surrounding `|tab-page|`.